tests/loggers/test_all.py

```diff
@@ -404,4 +405,27 @@ def test_logger_with_prefix_all(tmpdir, monkeypatch):
         wandb.run = None
         wandb.init().step = 0
         logger.log_metrics({"test": 1.0}, step=0)
-        logger.experiment.log.assert_called_once_with({'tmp-test': 1.0, 'trainer/global_step': 0})
+        logger.experiment.log.assert_called_once_with({'tmp-test': 1.0}, step=0)
+
+
+@pytest.mark.parametrize("close", [True, False])
+@pytest.mark.parametrize("logger_class", [
+    CometLogger,
+    CSVLogger,
+    MLFlowLogger,
+    NeptuneLogger,
+    TensorBoardLogger,
+    WandbLogger,
+])
+def test_logger_close_figure_all(logger_class, close, tmpdir):
+    f = plotting.dummy_figure()
+
+    logger = _instantiate_logger(logger_class, save_idr=tmpdir)
+
+    with mock.patch('matplotlib.pyplot.close') as plt_close:
+        logger.log_figure('dummy', f, 0, close=close)
+
+    if close:
+        plt_close.assert_called_once_with(f)
+    else:
+        plt_close.assert_not_called()
```

Comment:
Can someone help me out here? Both on my local machine as well as in the CI pipeline this tests fails for comet and MLFlow (even though both of them are installed).

Digging a bit into mlflow it says: 
```
    from .model import Model
E   ModuleNotFoundError: No module named 'mlflow.model'
```

and for comet I get
```
raise ImportError(msg)
E               ImportError: You must import Comet before these modules: torch, tensorboard
```

Is this expected? Of course I could wrap this part in a `try except ImportError` block but I do actually want to test the loggers ^^